src/tests/api/test_local_api.py

```diff
@@ -66,6 +66,53 @@ def setUp(self) -> None:
             'network': {}
         }
 
+        self.mock_application_json = [
```

Comment:
Would be great if the default application we create (qne application create) was valid python. Now we get an error after creating the application and immediate validating it because the main doesn't have the x and y parameter for the apps.

Reply:
Fixed